File: app/src/main/java/com/attendance735/attendonb/ui/vacation/pending/model/PendingResponse.java
package com.attendance735.attendonb.ui.vacation.pending.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class PendingResponse {
    @SerializedName("status")
    @Expose
    public Boolean status;
    @SerializedName("data")
    @Expose
    public PendingResponseData data;
    @SerializedName("code")
    @Expose
    public String code;
}

File: app/src/main/java/com/spidersholidays/attendonb/ui/vacation/pending/model/PendingResponse.kt
package com.spidersholidays.attendonb.ui.vacation.pending.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class PendingResponse {
    @SerializedName("status")
    @Expose
    var status: Boolean = false
    @SerializedName("data")
    @Expose
    var data: PendingResponseData? = null
    @SerializedName("code")
    @Expose
    var code: String? = null
}
